js/hitbtc3.js

```diff
@@ -1497,6 +1498,67 @@ module.exports = class hitbtc3 extends Exchange {
         };
     }
 
+    async fetchFundingRateHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        const request = {
+        // all arguments are optional
```

Comment:
These have to be indented. If you run `npm build` you should see that you get errors like `./ccxt/hitbtc3.py:1427:9: E122 continuation line missing indentation or outdented` and it's because comments inside the dictionary need to be indented